src/js/shariff.js

```diff
@@ -112,6 +112,12 @@ Shariff.prototype = {
 
         flattrUser: null,
 
+        createLinkIcon: function($shareLink, service) {
+            if (typeof service.faName !== 'undefined') {
+                $shareLink.prepend('<span class="fa ' +  service.faName + '">');
```

Comment:
The `<i>` is recommended for font awesome, also the `<span>` tag *is more semantically correct* 😕 
http://fontawesome.io/examples/